src/main/scala/com/github/unchama/seichiassist/menus/home/HomeMenu.scala

```diff
@@ -0,0 +1,366 @@
+      Button(
+        new IconItemStackBuilder(Material.COMPASS)
+          .title(s"$YELLOW$UNDERLINE${BOLD}ホームポイント${homeNumber}にワープ")
+          .lore(
+            List(
+              s"${GRAY}あらかじめ設定した",
+              s"${GRAY}ホームポイント${homeNumber}にワープします",
+              s"${DARK_GRAY}うまく機能しない時は",
+              s"${DARK_GRAY}再接続してみてください",
+              s"$DARK_RED${UNDERLINE}クリックでワープ",
+              s"${DARK_GRAY}command->[/home warp $homeNumber]"
+            )
+          )
+          .build(),
+        LeftClickButtonEffect {
+          SequentialEffect(
+            FocusedSoundEffect(Sound.BLOCK_STONE_BUTTON_CLICK_ON, 1f, 1f),
+            CommandEffect(s"home warp $homeNumber")
+          )
+        }
+      )
+
+    def setHomeButton(homeNumber: Int)(implicit environment: Environment): Button =
+      Button(
+        new IconItemStackBuilder(Material.BED)
+          .title(s"$YELLOW$UNDERLINE${BOLD}ホームポイント${homeNumber}を設定")
+          .lore(
+            List(
+              s"${GRAY}現在位置をホームポイント$homeNumber",
+              s"${GRAY}として設定します",
+              s"$DARK_GRAY※確認メニューが開きます",
+              s"$DARK_RED${UNDERLINE}クリックで設定",
+              s"${DARK_GRAY}command->[/home set $homeNumber]"
+            )
+          )
+          .build(),
+        LeftClickButtonEffect {
+          SequentialEffect(
+            FocusedSoundEffect(Sound.BLOCK_FENCE_GATE_OPEN, 1f, 0.1f),
+            environment.ioCanOpenConfirmationMenu.open(HomeChangeConfirmationMenu(homeNumber))
+          )
+        }
+      )
+
+    def removeHomeButton(homeNumber: Int)(implicit environment: Environment): Button =
+      Button(
+        new IconItemStackBuilder(Material.WOOL, 14)
+          .title(s"$RED$UNDERLINE${BOLD}ホームポイント${homeNumber}を削除")
+          .lore(
+            List(
+              s"${GRAY}ホームポイント${homeNumber}を削除します。",
+              s"$DARK_GRAY※確認メニューが開きます。",
+              s"$DARK_RED${UNDERLINE}クリックで設定"
+            )
+          )
+          .build(),
+        LeftClickButtonEffect {
+          FocusedSoundEffect(Sound.BLOCK_ENDERCHEST_CLOSE, 1f, 0.1f)
+          SequentialEffect(
+            environment
+              .ioCanOpenHomeRemoveConfirmationMenu
+              .open(HomeRemoveConfirmationMenu(homeNumber))
+          )
+        }
+      )
+  }
+}
+case class HomeMenuButtonComputations(player: Player) {
+  def setHomeNameButton[F[_]: HomeReadAPI: ConcurrentEffect](homeNumber: Int): IO[Button] = {
+    import cats.implicits._
+
+    val homeId = HomeId(homeNumber)
+
+    val program = for {
+      homeOpt <- HomeReadAPI[F].get(player.getUniqueId, homeId)
+    } yield {
+      val lore = homeOpt match {
+        case None => List(s"${GRAY}ホームポイント$homeId", s"${GRAY}ポイント未設定")
+        case Some(Home(optionName, location)) =>
+          val worldName = {
+            ManagedWorld
+              .fromName(location.worldName)
+              .map(_.japaneseName)
+              .getOrElse(location.worldName)
+          }
```

Comment:
```suggestion
          val worldName = 
            ManagedWorld
              .fromName(location.worldName)
              .map(_.japaneseName)
              .getOrElse(location.worldName)
```
Scalafmtで入ったものでなければいらないと思います

Reply:
ここは変更していない箇所で、scalafmtでもこのブレースは取れていませんでした。
ブレースを消しておきます。